core-services/ci-secret-generator/_config.yaml

```diff
@@ -105,3 +105,13 @@
     - app.ci
     service_account:
     - pod-scaler
+- item_name: endurance_cluster
+  attachments:
+  - cmd: |-
+      oc --context $(cluster) sa create-kubeconfig --namespace bparees $(service_account) | sed "s/$(service_account)/$(cluster)/g"
+    name: sa.$(service_account).$(cluster).config
+  params:
+    cluster:
+    - app.ci
+    service_account:
+    - endurance
```

Comment:
There is no serviceaccount `endurance` in the `bpaares` namespace

Reply:
because this PR isn't merged yet.  this PR introduces it:

https://github.com/openshift/release/pull/18643/files#diff-281a311b912a90db50c0eb522021bd39878c894f14eb07ae5b24b49968832530R1

does it need to exist first?  do i need to split the PR?

Reply:
Ah ok, that is fine. The failing presubmit is not, but it _should_ pass and seems like a bug on our end, I'll look into it